iocell/src/main/scala/chisel/IOCell.scala

```diff
@@ -63,76 +63,68 @@ class DigitalInIOCellBundle extends Bundle {
   val ie = Input(Bool())
 }
 
-abstract class IOCell extends BlackBox
+trait IOCell extends BaseModule
```

Comment:
Why does this need to be a `BaseModule`? The only way to drive `Analog` pins is a `BlackBox`.

Reply:
FireSim's IOCell models cannot be blackboxes